docs/TOKENS.md

```diff
@@ -162,14 +182,33 @@ Additional branded values ("secondary", "tertiary", "accent", "supports", etc) c
 
 Defines text highlight values
 
-### Forms
+### Forms <a name="tier-2-forms"></a>
 
 Defines the border width, radius, and color properties for form controls. These values are all mapped to other Tier 2 values and are considered optional in case remapping is required.
 
-### Shadows
+### Shadows <a name="tier-2-shadows"></a>
 
 - `box-shadow` defines the shadow value that gets applied by default to buttons and other "elevated" components like cards.
 
 ## Tier 3 component variables
 
 Tier 3 component variables represent each component's specific values.
+
+## Tailwind Class Tokens
+
+If the EDS tailwind config theme is being used, Tier 2 and tier 3 color tokens are available as a part of tailwind utility classes. Background, border, and text colors will only be available for themselves specifically. e.g.:
+
+```jsx
```

Comment:
Some might be confused with mixing classes that work or do not work (recommendations versus warnings). I'd recommend a "do/don't" set up here showing a div with allowed classes, and rationale, and ones that don't work and why (possibly as a header)

Reply:
Let me punt that into another PR, have to think a little more about finagling tables haha